README.md

```diff
@@ -9,7 +8,7 @@ Introduction
 -------------
 Genisys 3 is one of several derived projects resulting from the inactivity of **[PocketMine/PocketMine-MP](https://github.com/PocketMine/PocketMine-MP)** & **[ITX Tech](https://github.com/iTXTech/Genisys)** which is now commonly known as **[pmmp/PocketMine-MP](https://github.com/pmmp/PocketMine-MP)**. It has many gameplay features implemented which PocketMine-MP does not since Minecraft PE v0.11.
 
-[Genisys](https://github.com/iTXTech/Genisys/) has officially died and [FrontierDevs](https://github.com/FrontierDevs) are to revive it under the name Genisys3 
+[Genisys](https://github.com/iTXTech/Genisys/) has officially died and [FrontierDevs](https://github.com/FrontierDevs) are trying to revive it under the name Genisys3 
```

Comment:
@RTGDaCoder
[Genisys](https://github.com/iTXTech/Genisys/) has officially died and [FrontierDevs](https://github.com/FrontierDevs) are reviving it under the name Genisys3

Emm than, what about this? Is it okay?